tests/source/control-brace-style-always-next-line.rs

```diff
@@ -41,4 +41,18 @@ fn main() {
             val2
         },
     };
+
+    unsafe
+    {
+        let good = ();
+    }
+
+    unsafe {}
+
```

Comment:
I think there's a few comment-related cases not handled here, could you take a look at incorporating them as well?

```rust
    unsafe   /* f   */ {}
    unsafe /* f*/ {
        let x = 1;
    }
```

Reply:
I'll check those as well

Reply:
Any preference on the test file naming ?

Reply:
Great question, though looks like you already figured it out!